src/main/java/uk/gov/hmcts/reform/civil/service/CoreCaseDataService.java

```diff
@@ -153,4 +156,14 @@ public LocalDate getAgreedDeadlineResponseDate(Long caseId, String authorization
         }
         return null;
     }
+
+    public SearchResult getAllCases(String authorization) {
+        String query = mapper.createObjectNode()
```

Comment:
Perhaps we could use elastic search query builder instead of json mapper?

Reply:
completed with your help : )